scripts/modules/mozu-quotes-grid/mozuquotesgrid-view.js

```diff
@@ -0,0 +1,95 @@
+define(["modules/jquery-mozu",
+    "underscore",
+    "modules/backbone-mozu",
+    "modules/views-paging"],
+    function ($, _, Backbone, PagingViews) {
+
+        var mozuQuotesGridView = Backbone.MozuView.extend({
+            templateName: 'modules/mozuquotesgrid/grid',
+            initialize: function () {
+                var self = this;
+                var allowedToGetData = (this.model.hasRequiredBehavior() || !this.model.requireBehaviorsToRender);
+                if (allowedToGetData) {
+                    if (typeof this.model.apiGridRead === 'function') {
+                        this.model.apiModel.get = this.model.apiGridRead.bind(this.model);
+                        this.model.apiModel.setIndex = this.model.apiGridRead.bind(this.model);
+                    }
+
+                    try {
+                        this.$el.addClass('is-loading');
+                        if (this.model.get('autoload')) {
+                            self.model.setIndex(0);
+                        }
+                    } catch (error) {
```

Comment:
Why an empty try/catch block? This is bad practice and should be removed.

Reply:
removed try-catch